src/offices/_single.html

```diff
@@ -26,12 +28,14 @@
         {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
         <div class="content-l_col
                         {% if show_subscription -%}
```

Comment:
Should this be shortened to `{{ 'content-l_col-1-1' if show_subscription else 'content-l_col-1` }}` to match similar blocks elsewhere in the project?